### PR TITLE
Pwman3 distutils enable tests

### PR DESCRIPTION
**Pull request by oz123**
Closes: https://bugs.gentoo.org/927511

### CODE REVIEW

File: app-admin/pwman3/pwman3-0.12.2.ebuild [hunk 1]
```diff
@@ -24,8 +25,13 @@ RDEPEND="
 "
 BDEPEND="test? ( dev-python/pexpect[${PYTHON_USEDEP}] )"
 
+
 distutils_enable_tests setup.py
 
+python_test() {
+	python -m tests.test_pwman
```
Comment (thesamesam) [inline app-admin/pwman3/pwman3-0.12.2.ebuild:32]: This is missing a || die, and should be using ${EPYTHON}.

File: app-admin/pwman3/pwman3-0.12.2.ebuild [hunk 2]
```diff
@@ -24,8 +25,13 @@ RDEPEND="
 "
 BDEPEND="test? ( dev-python/pexpect[${PYTHON_USEDEP}] )"
 
+
 distutils_enable_tests setup.py
```
Comment (thesamesam) [inline app-admin/pwman3/pwman3-0.12.2.ebuild:29]: This also needs to actually be removed...

File: app-admin/pwman3/pwman3-0.12.2.ebuild (hunk 1 above)
Comment (oz123) [reply app-admin/pwman3/pwman3-0.12.2.ebuild]: fixed.

File: app-admin/pwman3/pwman3-0.12.2.ebuild (hunk 2 above)
Comment (oz123) [reply app-admin/pwman3/pwman3-0.12.2.ebuild]: oops ...

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**